integration-tests/features/stack_analyses_v2_valid_dynamic.feature

```diff
@@ -0,0 +1,164 @@
+      And I should find lxml with 4.2.1 4.5.1 4.5.0 with SNYK-PYTHON-LXML-72651 and 6.5 for public vulnerbilities
+      And I should find Cross-site Scripting (XSS) and medium for lxml in private vulnerbilities
+      And I should find License BSD for lxml in analyzed dependencies
+      And I should find wtforms with 2.1 2.3.1 2.3.1 with SNYK-PYTHON-WTFORMS-40581 and 6.5 for private vulnerbilities
+      And I should find License BSD for wtforms in analyzed dependencies
+      And I should find Cross-site Scripting (XSS) and medium for wtforms in private vulnerbilities
+  
+
+ @sav2 
+ Scenario: Data validation checks for npm ecosystem
+    Given System is running
+    Given Three scale preview service is running
+    When I acquire the use_key for 3scale
+    Then I should get the proper user_key
+    When I start npm package request with manifest valid_npmlist.json to stack analysis v2 transitives enabled with valid user key
+    Then I should get 200 status code
+     And I should receive JSON response containing the status key
+     And I should receive JSON response containing the id key
+     And I should receive JSON response containing the submitted_at key
+     And I should receive JSON response with the status key set to success
+     And I should receive JSON response with the correct id
+     And I should receive JSON response with the correct timestamp in attribute submitted_at
+    When I wait for stack analysis v2 to finish with user key
+    Then I should get stack analyses v2 response with all attributes
+      And I should find 3 unknown licenses for stack analyses v2
+      And I should find 8 distinct licenses for stack analyses v2
+      And I should find distinct license Artistic-2.0 for license analysis
+      And I should be able to validate github data for all dependencies
+      And I should get 16 transitive dependencies for serve-static
+      And I should get 6 transitive vulnerabilities for serve-static
+      And I should find bootstrap with 4.1.1 5.0.0-alpha1 4.5.0 with SNYK-JS-BOOTSTRAP-73560 and 6.5 for public vulnerbilities
+      And I should find Cross-site Scripting (XSS) and medium for bootstrap in public vulnerbilities
+      And I should find License MIT for bootstrap in analyzed dependencies
+      And I should find buefy with 0.7.10 0.8.20 0.8.19 with SNYK-JS-BUEFY-567814 and 7.3 for private vulnerbilities
+      And I should find Cross-site Scripting (XSS) and high for buefy in private vulnerbilities
+      And I should find License MIT for buefy in analyzed dependencies
+      
+
+
+ @sav2 
+ Scenario: Data validation checks for maven ecosystem
+    Given System is running
+    Given Three scale preview service is running
+    When I acquire the use_key for 3scale
+    Then I should get the proper user_key
+    When I start maven package request with manifest vertx_dependencies.txt to stack analysis v2 transitives enabled with valid user key
```

Comment:
I think new function we have is "I Send <pck> package .... ' function to send stack analyses request. Does this have some other flow?

Reply:
okay will change to same function